src/ec/packages/ec-component-card/card.story.js

```diff
@@ -0,0 +1,112 @@
+import { storiesOf } from '@storybook/html';
+import { withKnobs, text, array } from '@storybook/addon-knobs';
+import { withNotes } from '@ecl-twig/storybook-addon-notes';
+import withCode from '@ecl-twig/storybook-addon-code';
+
+import defaultSprite from '@ecl/ec-resources-icons/dist/sprites/icons.svg';
+import cardDocs from './docs/card.md';
+
+import card from './card.html.twig';
+
+storiesOf('Components/Card', module)
+  .addDecorator(withKnobs)
+  .addDecorator(withCode)
+  .addDecorator(withNotes)
+  .add(
+    'card',
+    () => {
+      const tagsArray = array(
+        'Tags (comma separated)',
+        ['Tag 1', 'Tag 2', 'Tag 3'],
+        ','
+      );
+      const infosArray = array(
+        'Infos (comma separated)',
+        ['2018/10/22', 'Luxembourg'],
+        ','
+      );
+      const metaArray = array(
+        'Meta (comma separated)',
+        ['Meta 1', 'Meta 2', 'Meta 3'],
+        ','
+      );
+      const tags = tagsArray.map(tag => ({
+        label: tag,
+        path: '/example',
+      }));
+
+      const infos = infosArray.map((info, key) => ({
+        label: info,
+        icon: {
+          type: 'general',
+          name: () => {
+            if (key === 0) {
+              return 'calendar';
+            }
+
+            if (key === 1) {
+              return 'location';
+            }
+
+            return 'faq';
+          },
+          path: defaultSprite,
+        },
+      }));
+
+      return card({
+        card: {
+          title: {
+            type: 'standalone',
+            path: text('Title path', '/example'),
```

Comment:
there is no real point in letting the user change the href of the link (no visual impact).
So I'd just put as a simple string

Reply:
Fixed in https://github.com/ec-europa/ecl-twig/pull/21/commits/78442bbb33e4a3550799ccb38601a6fc25482e3b